source/assets/stylesheets/index.scss

```diff
@@ -78,3 +78,17 @@
     }
   }
 }
+
+// This doesn't seem to be present -- it appears they removed it in one of the more recent releases.
+// Give me my rounded buttons, dammit!
+button.radius, .button.radius {
+    border-radius: 3px; }
+button.rounded, .button.rounded {
+  border-radius: 1000px; }
+
+@media screen and (max-width: 74.9375em) {
+  .button.large-down-expanded {
+    display: block;
+    width: 100%;
+    margin-right: 0;
+    margin-left: 0; } }
```

Comment:
A couple things here:

1. Always have a newline at the end of a file
2. Don't create new `@media` queries, it'll actually be more headache than it's worth, long-term.
3. Try to work within what Foundation offers as much as possible. The button size might be enlarged by increasing the font size within the buttons. Or do you want full blocks to click like what we do with the navigation bar at the top?

Reply:
All of the CSS added in this file is directly from the Foundation documentation as what to do to enable the selected features.